commit 8abb5896fcc035566a57cb49b4f27172d7b5b726
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-05-17T11:57:27-04:00 (5 months ago)
    
    TRITON-1660 32-bit registrar should be buildable on x86_64
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index afd4340..18b9bfc 100644
--- a/package.json
+++ b/package.json
@@ -11,14 +11,14 @@
     "dependencies": {
         "assert-plus": "0.1.5",
         "backoff": "2.4.0",
-        "bunyan": "0.23.1",
+        "bunyan": "1.8.12",
         "clone": "0.1.17",
         "dashdash": "1.6.0",
         "node-uuid": "1.4.0",
         "once": "1.3.0",
         "vasync": "1.5.0",
         "verror": "1.4.0",
-        "zkplus": "git://github.com/mcavage/node-zkplus.git#master"
+        "zkplus": "git+https://github.com/mcavage/node-zkplus.git#7a42e61197fce303b38dd8779c1894a0f3b5d00c"
     },
     "devDependencies": {
         "faucet": "0.0.1",
